<launch>
  <node name="navigation" pkg="we_navigation" type="we_navigation" output ="screen">
    <rosparam file="$(find we_navigation)/cfg/navigation.yaml" command="load" />

    <param name="nodeFilePath" value="$(find we_navigation)/map/nodes.txt"/>
    <param name="edgeFilePath" value="$(find we_navigation)/map/edges.txt"/>
  </node>
</launch>

